.travis.yml

```diff
@@ -1,7 +1,7 @@
 language: node_js
 node_js:
-  - "8.0"
-before_script: "npm run validate"
+  - "8"
+  - latest
```

Comment:
Let's just target 8.9.0 (LTS)

Reply:
This was actually wrong, it should be `"node"`

I don't see why testing against the current node version is a bad idea

Reply:
It's not a terrible idea. But it's better for the project overall to have a single target node version which it supports and LTS is perfect for that. I'd rather avoid surprise breaks in tests.

Reply:
sure, current might break, but the chance is very low and there's still a lot of people running it locally

Reply:
If it does break because of the node version that should be rather easy to detect